### PR TITLE
created iff2 function for numpy frontend

### PR DESCRIPTION
**Pull request by EugeneB333**
Please find this PR for issue #1532 on Fourier Transform for numpy frontend.

### CODE REVIEW

**Review by zaeemansari70 — COMMENTED** [review]
Hi @EugeneB333 ,

I've seen you have not yet added the test for the function you are working on, do let me know by commenting that you are done with your implementation so I can review your PR.

Do let me know if you have any other questions!
Thanks! :slightly_smiling_face:

**Comment by EugeneB333** [comment]
> Hi @EugeneB333 ,
> 
> I've seen you have not yet added the test for the function you are working on, do let me know by commenting that you are done with your implementation so I can review your PR.
> 
> Do let me know if you have any other questions! Thanks! slightly_smiling_face

Added the test! Sorry, should have done this earlier.

File: ivy/functional/frontends/numpy/fft/discrete_fourier_transform.py [hunk 1]
```diff
@@ -8,3 +8,11 @@ def ifft(a, n=None, axis=-1, norm=None):
...
         norm = "backward"
     return ivy.ifft(a, axis, norm=norm, n=n)
+    
+    
+@to_ivy_arrays_and_back
+def ifft2(a, s=None, axis=(-2, -1), norm=None):
+    a = ivy.array(a, dtype=ivy.complex128)
+    if norm is None:
+        norm = "backward"
+    return ivy.ifft2(a, axis, norm=norm, s=s)
```
Comment (sherry30) [inline ivy/functional/frontends/numpy/fft/discrete_fourier_transform.py:18]: ivy API does not have `ifft2` you will have to use `ivy.ifft`

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**